tensorflow/lite/kernels/embedding_lookup_test.cc

```diff
@@ -116,6 +114,33 @@ TEST(EmbeddingLookupOpTest, SimpleTest) {
               })));
 }
 
+TEST(EmbeddingLookupOpTest, IndexTooHighTest) {
+  EmbeddingLookupOpModel m({3}, {3, 2, 4});
+  m.SetInput({3, 0, 2});
```

Comment:
Keep the test name as is, but you only need m.SetInput({3}) to test the index too high failure. All of the other inputs don't add to testing the failure behavior

Reply:
There is only one behavior we want to test (failing on the index being too high). This is currently being tested in both "IndexTooHighTest" and "SingleIndexTest". We should dedup this.

Reply:
@alanchiao thanks for the feedback, i have updated based on your comments.

Regards
Amit